docs/pages/upcoming-releases.mdx

```diff
@@ -57,7 +35,7 @@ The key deliverables for Teleport Cloud in the next quarter:
 
 | Week of            | Description                                                |
 |--------------------|------------------------------------------------------------|
-| October 30th, 2023 | Teleport 14.1 will begin rollout                           |
+| October 20th, 2023 | Teleport 14.2 will begin rollout                           |
```

Comment:
This is Thanksgiving week so it is a bit of a risk to roll out. We can upgrade on Monday or Tuesday if needed but the safest thing would be to wait until the following week. Up to you.

Reply:
Shouldn't this say November 20?